rpcsx-os/iodev/shm.cpp

```diff
@@ -40,6 +40,9 @@ orbis::ErrorCode ShmDevice::open(orbis::Ref<orbis::File> *file,
     } else if (name == "/rpcsx-SceNpPlusLogger") {
       realFlags |= O_CREAT;
       size = 0x4400;
+    } else if (name == "/rpcsx-SceAvSetting") {
+      realFlags |= O_CREAT;
+      size = 0x4400;
```

Comment:
```suggestion
    } else if ((name == "/rpcsx-SceNpPlusLogger") || (name == "/rpcsx-SceAvSetting")) {
      realFlags |= O_CREAT;
      size = 0x4400;
    }
```

no need two if for same setting set